sdk/src/account.rs

```diff
@@ -47,7 +47,7 @@ mod account_serialize {
         lamports: u64,
         #[serde(with = "serde_bytes")]
         // a ref so we don't have to make a copy just to serialize this
-        data: &'a Vec<u8>,
+        data: &'a [u8],
```

Comment:
you can get rid of the 'data' parameter below and use account.data()

Reply:
yup. fixed.